src/client/lib/url.ts

```diff
@@ -0,0 +1,15 @@
+class Url {
+    get login(): string {
+        return `/login?${window.location.search}`;
+    }
+
+    get configurationCreate(): string {
+        return `/configuration/create?${window.location.search}`;
+    }
+
+    get configurationUpdate(): string {
+        return `/configuration/update?${window.location.search}`;
+    }
```

Comment:
I think we should prefer being explicit for query parameters while building these URLs.  

As a person reading this could, I should be able to understand what parameters can be passed to these URLs and if they are required or not. Otherwise it gets obfuscated and hard to follow, hence gets more error prone.